README.md

```diff
@@ -26,4 +26,73 @@ $ solana sign-offchain-message --keypair <keypair.json> "hey there"
 
 $ solana verify-offchain-signature --signer 4fwzf5RUcptqpMTYjNhXVuYabeX9bF1patCSYTkMufFN "hey there" 3zHMng1qyLHbPTBw1YxBVwJGBWhkqRrX6JipKqmWiT9P4oMEuacqdjvWEbHaG6CbMF6rYL4e22qkP7zcY8FP4Bfr
 Signature is valid
-```
\ No newline at end of file
+```
+
+### Celestia
+The [celestia-app](https://docs.celestia.org/how-to-guides/celestia-app-commands) CLI does not provide a way to sign arbitrary message out of the box.
```

Comment:
i guarantee you at least one dipshit will read this sentence only and quit, then message harry about how celestia can't do it.

you gatta follow this statement up with some shit like -- `SO KEEP READING MOTHERFUCKER` well, maybe something nicer like -- , so follow the instructions below in order to sign (what are we having them sign anyway?)